frontend/packages/console-app/src/components/volume-snapshot/volume-snapshot-details.tsx

```diff
@@ -69,13 +73,17 @@ const Details: React.FC<DetailsProps> = ({ obj }) => {
                 </dd>
               </>
             )}
-            <dt>Volume Snapshot Class</dt>
-            <dd data-test="details-item-value__SC">
-              <ResourceLink
-                kind={referenceForModel(VolumeSnapshotClassModel)}
-                name={obj?.spec?.volumeSnapshotClassName}
-              />
-            </dd>
+            {snapshotClass && (
```

Comment:
Are we sure we want to omit the entire Volume Snapshot Class detail item? Should we use an empty state value 'None' or '-'? That way the user can see that this concept exists even if it's not defined for this instance.